modules/mukurtu_core/mukurtu_core.links.menu.yml

```diff
@@ -5,13 +5,13 @@ mukurtu_core.dashboard:
   menu_name: mukurtu
   route_name: mukurtu_core.dashboard
   weight: -1000
-mukurtu_core.dashboard_main:
-  title: 'Dashboard'
-  description: 'Dashboard'
-  parent: main
-  menu_name: main
-  route_name: mukurtu_core.dashboard
-  weight: 1000
+#mukurtu_core.dashboard_main:
```

Comment:
I am of an opinion that all 'non-context-documenting' comments in yml files can safely be removed since we can capture before and after (and diff side-by-side) any changes but defer to you both if you like having historical context captured going forward

Reply:
In this case specifically, it's removing something that wasn't intended to be there in the first place, so I don't think it's a change that needs to be retained like that.

Reply:
Sounds good to me I am going to pull this back to a draft and look at the other two items that are commented out and see if it makes sense just to remove those as well...